src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   PATTERN "frontend/*" EXCLUDE
   PATTERN ".*" EXCLUDE)
 
+include(ExternalProject)
+ExternalProject_Add(grafonnet-lib-0.1.0
```

Comment:
please do not encode the version in the target name unless the version really matters like python2 and python3.

how is this target integrated into the build process?

Reply:
@tchaikov I was initially not keen on pinning deps versions, but with [the increasing number of malware being injected into code or package repos](https://www.bleepingcomputer.com/news/security/pypi-packages-caught-stealing-credit-card-numbers-discord-tokens/), I think version pinning is the simplest way to avoid [supply chain attacks](https://www.wired.com/story/hacker-lexicon-what-is-a-supply-chain-attack/). This is the same we are doing with the Github Actions (pinning to specific commits we have manually reviewed).

Reply:
@epuertat could you please re-read my comment?

Reply:
Ok, your concern was about adding the version IN the target name, not about the version pinning itself, right?

Reply:
right.